src/components/external-link.tsx

```diff
@@ -13,7 +13,7 @@ export function ExternalLink({
     <a
       href={href}
       target="_blank"
-      rel="noopener"
+      rel="noopener noreferrer"
```

Comment:
`noreferrer` makes it so the other website doesn't know where the user came from, which we have had issues with in the past.
```suggestion
      rel="noopener"
```